core/templates/dev/head/pages/exploration-player-page/services/question-player-engine.service.ts

```diff
@@ -157,12 +156,10 @@ angular.module('oppia').factory('QuestionPlayerEngineService', [
        *   - initHtml {string}, an HTML string representing the content of the
        *       first state.
        */
-      init: function(questionDicts, successCallback) {
+      init: function(questionObjects, successCallback) {
```

Comment:
Ditto* (here and elsewhere)

Reply:
Should this be renamed to `Questions`? There is already a local var `questions`.

Reply:
done